tests/integration/istioio/trafficmanagement/trafficshifting/main_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package circuitbreaker
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+)
+
+var (
+	inst istio.Instance
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("trafficshifting", m).
+		SetupOnEnv(environment.Kube, istio.Setup(&inst, nil)).
```

Comment:
This isn't doing any special environment setup here. Can we move the test to  tests/integration/istioio/trafficmanagement/? If we have the test in its own directory, it will spin up a brand new environment just for the one test.

Reply:
Done. Moved it to trafficmanagement directory from its own directory.